packages/browser/src/Component/Widget/C3DTiles.js

```diff
@@ -0,0 +1,174 @@
+import * as itownsWidget from 'itowns/widgets';
+import * as itowns from 'itowns';
+import { createLabelInput } from '../HTMLUtil';
+import * as THREE from 'three';
+
+const DEFAULT_OPTIONS = {
+  position: 'top-right', // should be deprecated https://github.com/iTowns/itowns/issues/2005
+};
+
+export class C3DTiles extends itownsWidget.Widget {
```

Comment:
This make me wonder : As we are now really close to Itowns, should we not always create widget extending itownsWidget ?

Reply:
yes we should to make a possible contribution easier then, even if this class is quite doing nothing.. I tried with this widget to illustrate how should looks like a widget implementation.